scripts/masterCron.ts

```diff
@@ -581,15 +589,17 @@ async function strikeForecasting() {
 }
 
 async function commitAndClose() {
+  console.log("Calling commitAndClose");
+
   const vaultArtifact = await hre.artifacts.readArtifact("RibbonThetaVault");
   const otokenArtifact = await hre.artifacts.readArtifact("IOtoken");
 
   // 1. commitAndClose
   await runTX(vaultArtifact.abi, provider, signer, network, "commitAndClose");
 
-  await sleep(10000);
+  await sleep(TX_SLEEP_TIME);
```

Comment:
you think 5 minutes is enough for aave/eth/steth/btc all confirming sequentially with 20gwei above standard?